### ISSUE TITLE
Can't initialize repository

### ISSUE THREAD

**Issue by chase-dwelle** [vermiculus/magithub#55]
I saw there was another issue for this problem, but I could not resolve it for myself. I am fairly new to emacs.

Whenever I try to add an issue to a repository, I get the error `Hub hasn't been initialized yet; aborting`. I have run `hub browse` within the terminal (using both my terminal application and `M-x term` inside emacs).

Is there something simple that I am missing here? Running `hub version` gives me
`git version 2.6.3`
`hub version 2.2.2`

**Comment by vermiculus** [comment]
What does `hub version` give you when you run it from `M-x eshell` or `M-!`?

A very common problem with emacs newcomers is a difference in the PATH environment variable between the emacs instance and your login/interactive shell. I would try using [`exec-path-from-shell`](https://github.com/purcell/exec-path-from-shell) to see if that helps.

**Comment by chase-dwelle** [comment]
I fixed the issue, which was caused by the fact that I did not have a `~/.config/hub`. For others that may run into this issue, I don't see a way to set up the `hub` configuration, so I just cloned one of my private repositories using `hub clone [username]/[repo_name]`, which prompted me for my username/password to create the OAUTH token and store this info in `~/.config/hub`.

Now I don't have any problems when going to create issues in my repos and don't receive any `Hub hasn't been initialized yet; aborting' errors. Thanks for the quick feedback!
